azure_monitor/azfunc_sample/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "This sample contains a simple Azure Function to show how you can send correlated application log information as well as information of external dependencies to Azure Monitor (Application Insights)."
+urlFragment: azure-monitor-opencensus-python
+---
+
+# Correlated End-To-End Azure Function (Python) Instrumentation with Opencensus 
+
+## Overview
+
+Azure Function (Python) already supports instrumentaton using Opencensus. Unfortunately, the current out-of-the-box (OOTB) implementation/support has some flaws (see below), which make it difficult to get a correlated end-to-end view of a single function invocation (in Azure Monitor/Application Insights).
```

Comment:
I would not say "flaws" nor "unfortunately", as this refers to a bug of some kind. Maybe reword this to: `The following sample utilizes the OpenCensus Functions extension and enables additional common telemetry scenarios used by Azure Monitor and Functions users. Such scenarios include:...`

Reply:
done. updated.